browser_use/browser/session.py

```diff
@@ -710,6 +697,7 @@ async def setup_browser_via_browser_pid(self) -> None:
 		# we could automatically relaunch the browser process with that arg added here, but they may have tabs open they dont want to lose
 		self.cdp_url = self.cdp_url or f'http://localhost:{debug_port}/'
 		self.logger.info(f'🌎 Connecting to existing local browser process: browser_pid={self.browser_pid} on {self.cdp_url}')
+		assert self.playwright is not None, 'playwright instance is None'
```

Comment:
why would it ever be none at this point? this is not called until after playwright setup